src/index.js

```diff
@@ -0,0 +1,63 @@
+const state = {
+    tempCount : 0
+};
+
+const changeTempColor = () => {
+    if (state.tempCount <= 49) {
+        document.getElementById("tempCounter").style.color = "teal";
+    } else if (state.tempCount <= 59) {
+        document.getElementById("tempCounter").style.color = "green";
+    } else if (state.tempCount <= 69) {
+        document.getElementById("tempCounter").style.color = "yellow";
+    } else if (state.tempCount <= 79) {
+        document.getElementById("tempCounter").style.color = "orange";
+    } else if (state.tempCount >= 80) {
+        document.getElementById("tempCounter").style.color = "red";
+    } 
+}
+
+const changeLandscape = () => {
+    if (state.tempCount <= 59) {
+        document.getElementById("landscape").textContent = "🌲🌲⛄️🌲⛄️🍂🌲🍁🌲🌲⛄️🍂🌲";
+    } else if (state.tempCount <= 69) {
+        document.getElementById("landscape").textContent = "🌾🌾_🍃_🪨__🛤_🌾🌾🌾_🍃";
+    } else if (state.tempCount <= 79) {
+        document.getElementById("landscape").textContent = "🌸🌿🌼__🌷🌻🌿_☘️🌱_🌻🌷";
+    } else if (state.tempCount >= 80) {
+        document.getElementById("landscape").textContent = "🌵__🐍_🦂_🌵🌵__🐍_🏜_🦂";
+    }
+}
+
+const changeSky = () => {
+    var value = selectObject.value;  
```

Comment:
There's no variable in scope here called `selectObject`, we would need to retrieve an instance of the sky element first if we wanted to be able to use its value
```js
    const selectObject = document.querySelector("#sky");
```